src/themis/secure_session_utils.c

```diff
@@ -190,7 +191,7 @@ themis_status_t verify_mac(const void *key, size_t key_length, const soter_kdf_c
 		return THEMIS_INVALID_PARAMETER;
 	}
 
-	if (memcmp(mac, computed_mac, mac_length))
+	if (cst_time_memcmp(mac, computed_mac, mac_length))
```

Comment:
@secumod is this the only valid check here? or mac is public too?

Reply:
Hm... Seems it is public as well